src/store/index.js

```diff
@@ -1,25 +1,8 @@
 import { createStore } from "vuex";
-import createPersistedState from "vuex-persistedstate";
-import SecureLS from "secure-ls";
 import auth from "./modules/auth";
 
-var localStorage = new SecureLS({ isCompression: false });
-
 export default createStore({
```

Comment:
This change should not be in this PR?

Reply:
We didn't want the secure-ls package anymore, so I deleted the code that uses it.